frontend/src/actions/taskActions.js

```diff
@@ -272,6 +290,30 @@ const updateTask = task => {
   }
 }
 
+const deleteTask = (task, history) => {
```

Comment:
We don't need to pass the history and we should call on the component straight we we return the axios call as a promise